shell/pkg-stats.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -ue
```

Comment:
at first glance, this looks like it could be `/bin/sh` as not using anything bash specific (makes it portable)

Reply:
I don't think `|&` is portable, although that could be trivially changed. Why not just use the standard `#!/usr/bin/env bash` and `set -ue` below... I know it'll force you to install `bash` on your OpenBSD boxes :wink: